src/templates/base-generated-config.yaml

```diff
@@ -11,18 +11,22 @@ jobs:
         type: string
       test-suite-path:
         type: string
+      bb-version:
+        type: string
     machine:
       image: ubuntu-2004:current
       docker_layer_caching: true
     steps:
-      - checkout
+      - run:
+          name: Checkout Specific Version
+          command: git checkout << parameters.bb-version >>
```

Comment:
Versioning shouldn't use release version as a whole as it might not contain candidate applications. It should rather take test suites from version and candidiate applications from current branch.

Reply:
Thanks for pointing this out, I will fix it